src/angular/src/app/spark-docs/input-docs/input-docs.component.ts

```diff
@@ -137,16 +137,22 @@ import { Component } from '@angular/core';
               #searchInput="ngModel"
               sprkInput>
           </sprk-input-container>
-          <sprk-input-container>
+          <sprk-icon-input-container iconContainerClasses="">
+            <label
+              class="sprk-b-Label--with-icon sprk-u-ScreenReaderText"
+              sprkLabel>
+              Search
+            </label>
+            <sprk-icon iconType="search" additionalClasses="sprk-b-InlineSearch__icon" sprk-input-icon></sprk-icon>
             <input
               name="inline_search_input"
-              type="search"
-              role="search"
+              class="sprk-b-TextInput--with-icon"
+              type="text"
```

Comment:
Previous version has `type="search"` and `role="search"`. Is there a reason for that change? Same in the docs.